### ISSUE TITLE
Outdated Testing Files

### ISSUE THREAD

**Issue by AnthonyRodway** [cat-cfs/twobilliontoolkit#22]
The unit tests in twobilliontoolkit are getting neglected with the other things going on, so at a lower priority it would be nice to catch those up and make it a bit easier to use.

**Comment by AnthonyRodway** [comment]
I will be roping in updating README's into this ticket as a general cleanup ticket for the repo. 

It was mentioned at one of our previous meetings about the idea of using something such as https://realpython.com/python-project-documentation-with-mkdocs/ for easily creating documentation

**Comment by AnthonyRodway** [comment]
Also for the new ticket https://github.com/cat-cfs/twobilliontoolkit/issues/34 I think they can be done somewhat in parallel

**Comment by AnthonyRodway** [comment]
The mkdocs has been implemented and the readmes updated to the current specifications. The only thing left for this ticket would be updating the testing files but that will come a bit later.
